service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/UpcomingHearingsFoundNotificationHandler.java

```diff
@@ -1,44 +0,0 @@
-package uk.gov.hmcts.reform.fpl.handlers;
-
-import lombok.RequiredArgsConstructor;
-import lombok.extern.slf4j.Slf4j;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.context.event.EventListener;
-import org.springframework.stereotype.Component;
-import uk.gov.hmcts.reform.fpl.config.CtscEmailLookupConfiguration;
-import uk.gov.hmcts.reform.fpl.events.UpcomingHearingsFound;
-import uk.gov.hmcts.reform.fpl.model.notify.NotifyData;
-import uk.gov.hmcts.reform.fpl.service.FeatureToggleService;
-import uk.gov.hmcts.reform.fpl.service.email.NotificationService;
-import uk.gov.hmcts.reform.fpl.service.email.content.UpcomingHearingsContentProvider;
-
-import static org.springframework.util.CollectionUtils.isEmpty;
-import static uk.gov.hmcts.reform.fpl.NotifyTemplates.UPCOMING_HEARINGS_TEMPLATE;
-
-@Slf4j
-@Component
-@RequiredArgsConstructor(onConstructor = @__(@Autowired))
-public class UpcomingHearingsFoundNotificationHandler {
```

Comment:
Do we need to remove the UpcomingHearingsFound.java, and the quartz job, UpcomingHearingsFinder.java as well? Also, the cron configuration in application.yaml.

It seems that the UpcomingHearingsFinder job is just searching for a upcoming hearing and then send a notification

Reply:
Yeah, think this job is only used to notify the CTSC admin just now.